tests/test_cli.py

```diff
@@ -114,7 +114,7 @@ def test_command_login(workdir, mocker):
     ('traintuple', 'key', models.Traintuple),
     ('aggregatetuple', 'key', models.Aggregatetuple),
     ('composite_traintuple', 'key', models.CompositeTraintuple),
-    ('compute_plan', 'compute_plan_id', models.ComputePlan),
+    ('compute_plan', 'key', models.ComputePlan),
```

Comment:
As for the `test_command_get `, the key_field parameter is now useless, could you remove it? I'm sorry I forgot to include this comment in my previous review.

Reply:
Fixed in https://github.com/SubstraFoundation/substra/pull/241/commits/7071f4b4f636edc6646124d22d4390062434ee93